content/02.main-text.md

```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
 These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
 GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
 
-`TODO: cgreene paragraph on diversity, open science culture, possibly author disagreements and conduct`
+It is important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
+Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1080/08874417.2009.11645338].
```

Comment:
`10.1080/08874417.2009.11645338` is not a registered DOI

Reply:
Interesting, this article is titled "The Role of Wiki Technology and Altruism in Collaborative Knowledge Creation" and was published in 2009 in the _Journal of Computer Information Systems_. The publisher's site is http://www.tandfonline.com/doi/abs/10.1080/08874417.2009.11645338, which states that the DOI is `10.1080/08874417.2009.11645338`, however this is not a registered DOI. In addition, the title is not registered with Crossref, although a precursor of the article does have a valid DOI: https://doi.org/10.1109/hicss.2007.277.

Reply:
Options are:

+ do not cite
+ cite with DOI, but manually override JSON CSL. Will result in broken DOI link that may resolve in the future
+ cite with URL
+ cite `@doi:10.1109/hicss.2007.277` instead

Reply:
I did not address this one. @agitter : what is your preference here?

Reply:
The solution proposed below looks good to me